src/scripts/componentsHandler.es6

```diff
@@ -1,4 +1,4 @@
-module.exports = function (components) {
+module.exports = (components) => {
```

Comment:
Tohle je, pravda, trochu věc názoru, ale myslím, že je lepší používat arrow functions, kdekoliv to jde. Jednak to pak zčitelní užití klasických functions (člověk hned uvidí, že se očekává `this` zvenčí), a taky to bude v kompilátorech dnů budoucích trochu rychlejší ‒ arrow functions jsou prostě navržené, aby byly lightweight. A v neposlední řadě prostě méně kódu = win.